packages/server/graphql/types/UpdatePromptResponsePayload.ts

```diff
@@ -0,0 +1,31 @@
+import {GraphQLObjectType} from 'graphql'
+import {GQLContext} from '../graphql'
+import {makeResolve, resolveNewMeeting} from '../resolvers'
+import makeMutationPayload from './makeMutationPayload'
+import NewMeeting from './NewMeeting'
+import StandardMutationError from './StandardMutationError'
+import TeamPromptResponse from './TeamPromptResponse'
+
+export const UpdatePromptResponseSuccess = new GraphQLObjectType<any, GQLContext>({
+  name: 'UpdatePromptResponseSuccess',
+  fields: () => ({
+    error: {
+      type: StandardMutationError
+    },
```

Comment:
-1 `makeMutationPayload` takes care of resolving to error type if the error is present so this isn't needed